example/deployment/terraform-modules/deploy_palisade/main.tf

```diff
@@ -0,0 +1,171 @@
+  provisioner "remote-exec" {
+    inline = [
+      "mkdir -p /home/${var.ec2_userid}/example/deployment/local-jvm/bash-scripts",
+      "mkdir -p /home/${var.ec2_userid}/example/deployment/bash-scripts",
+      "mkdir -p /home/${var.ec2_userid}/example_logs",
+      "mkdir -p /home/${var.ec2_userid}/example_data",
+      "mkdir -p /home/${var.ec2_userid}/example/example-services/example-rest-config-service/target",
+      "mkdir -p /home/${var.ec2_userid}/example/example-model/target",
+      "mkdir -p /home/${var.ec2_userid}/example/example-model/src/main/resources",
+      "mkdir -p /home/${var.ec2_userid}/example/example-services/example-rest-resource-service/target",
+      "mkdir -p /home/${var.ec2_userid}/example/example-services/example-rest-user-service/target",
+      "mkdir -p /home/${var.ec2_userid}/example/example-services/example-rest-policy-service/target",
+      "mkdir -p /home/${var.ec2_userid}/example/example-services/example-rest-palisade-service/target",
+      "mkdir -p /home/${var.ec2_userid}/example/example-services/example-rest-data-service/target",
+      "mkdir -p /home/${var.ec2_userid}/example/resources",
+    ]
+  }
+
+  # Copy bash-scripts directories to ec2 instance......and make scripts executable
+  provisioner "file" {
+    source = "../../../bash-scripts/"
+    destination = "/home/${var.ec2_userid}/example/deployment/bash-scripts"
+  }
+  provisioner "file" {
+    source = "../../bash-scripts/"
+    destination = "/home/${var.ec2_userid}/example/deployment/bash-scripts"
+  }
+  provisioner "file" {
+    source = "../../../local-jvm/bash-scripts/"
+    destination = "/home/${var.ec2_userid}/example/deployment/local-jvm/bash-scripts"
+  }
+  # Convert all dos line endings to unix
+    provisioner "remote-exec" {
+      inline = [
+        "dos2unix /home/${var.ec2_userid}/example/deployment/bash-scripts/*.sh",
+        "dos2unix /home/${var.ec2_userid}/example/deployment/local-jvm/bash-scripts/*.sh",
+      ]
+    }
+  provisioner "remote-exec" {
+    inline = [
+      "chmod 774 /home/${var.ec2_userid}/example/deployment/bash-scripts/*.sh",
+      "chmod 774 /home/${var.ec2_userid}/example/deployment/local-jvm/bash-scripts/*.sh",
+    ]
+  }
+  # Copy configRest.json to ec2 instance
+  provisioner "file" {
+    source = "../../../../example-model/src/main/resources/configRest.json"
+    destination = "/home/${var.ec2_userid}/example/example-model/src/main/resources/configRest.json"
+  }
+
+  provisioner "remote-exec" {
+    inline = [
+      "/home/${var.ec2_userid}/example/deployment/local-jvm/bash-scripts/stopAllServices.sh",
+      "/home/${var.ec2_userid}/example/deployment/local-jvm/bash-scripts/startETCD.sh",
+    ]
+  }
+
+  # Deploy the Palisade config service on the ec2 instance......1st copy over its jar....
+  provisioner "file" {
+    source = "../../../../example-services/example-rest-config-service/target/example-rest-config-service-${var.palisade_version}-SNAPSHOT-executable.jar"
```

Comment:
do a find and replace on this file "-${var.palisade_version}-SNAPSHOT-" to "-${var.palisade_version}-".
Otherwise this will not work on release versions